### PR TITLE
Project docs update

### PR DESCRIPTION
**Pull request by lehasm**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/project/__init__.py [hunk 1]
```diff
@@ -62,49 +62,50 @@
 
 
 class Project(BaseTolokaObject):
-    """Top-level object in Toloka. All other entities are contained in some project.
+    """Top-level object in Toloka that describes one requester's task.
```
Comment (alexandervnuchkov) [inline src/client/project/__init__.py:65]: I think, `task` is not exactly suitable word here, as we use tasks to denote the items we upload to the pool. Perhaps something like `goal` or `a set of tasks` will fit better here? Or maybe you have your own variant

File: src/client/project/__init__.py [hunk 2]
```diff
@@ -114,47 +115,48 @@ class Project(BaseTolokaObject):
...
 
         @unique
         class MapProvider(ExtendableStrEnum):
-            """Map provider for assignments_issuing_view_config:
+            """A map provider.
 
             Attributes:
-                YANDEX: Use Yandex Maps as a map provider
-                GOOGLE: Use Google Maps as a map provider
+                YANDEX: Yandex Maps.
```
Comment (alexandervnuchkov) [inline src/client/project/__init__.py:158]: Is it possible to reverse the order and make it alphabetic (first `GOOGLE`, then `YANDEX`)?

File: src/client/project/__init__.py [hunk 3]
```diff
@@ -194,32 +196,32 @@ def __attrs_post_init__(self):
...
         self.localization_config.default_language = language
 
     def add_requester_translation(self, language: str, public_name: Optional[str] = None, public_description: Optional[str] = None, public_instructions: Optional[str] = None):
-        """Add new translations to other language.
+        """Adds a project interface translation to other language.
 
-        You can call it several times for different languages.
-        If you call it for the same language, it overwrites new values, but don't overwrite values, that you don't pass.
+        To translate to several languages call this method for each translation.
+        Subsequent call with the same `language` updates the translation of passed parameters and doesn't touch omitted parameters.
```
Comment (alexandervnuchkov) [inline src/client/project/__init__.py:213]: `A subsequent call`

File: src/client/project/localization.py [hunk 4]
```diff
@@ -1,60 +1,55 @@
...
+
+
+class AdditionalLanguage(BaseTolokaObject):
+    """A translation of a project interface.
+
+    Args:
+        language: The language into which the translation is made. Two-letter [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code in upper case.
+        public_name: A translated project name.
+        public_description: A translated project description.
+        public_instructions: A translated instructions for Tolokers.
```
Comment (alexandervnuchkov) [inline src/client/project/localization.py:19]: No need to use the article (`instructions` - plural)

File: src/client/project/view_spec.py [hunk 5]
```diff
@@ -114,28 +113,26 @@ class Assets(BaseTolokaObject):
...
-        view:
-        plugins:
-        vars:
-        core_version: Default template components version. Most likely you do not need to change this parameter.
-        infer_data_spec: You can configure the data specification automatically or manually. You can change the way the
-            specification is configured using the infer_data_spec option:
+        view: A top level component like [SideBySideLayoutV1](toloka.client.project.template_builder.layouts.SideBySideLayoutV1.md).
+        plugins: An array of plugins.
+        vars: Reusable data. It is referenced with the [RefComponent](toloka.client.project.template_builder.base.RefComponent.md).
+        core_version: The default template components version. Most likely you do not need to change this parameter.
```
Comment (alexandervnuchkov) [inline src/client/project/view_spec.py:124]: `Most likely, you do not need to change this parameter.`

File: src/client/project/__init__.py (hunk 3 above)
Comment (lehasm) [reply src/client/project/__init__.py:213]: Fixed

File: src/client/project/localization.py (hunk 4 above)
Comment (lehasm) [reply src/client/project/localization.py:19]: Fixed

File: src/client/project/view_spec.py (hunk 5 above)
Comment (lehasm) [reply src/client/project/view_spec.py:124]: Thanks.

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**